internal/app/state.go

```diff
@@ -68,6 +68,15 @@ func (s *state) setDefaults() {
 	}
 }
 
+func (s *state) initializeNamespaces() {
+	for nsName, ns := range s.Namespaces {
+		if ns == nil {
+			ns = &namespace{}
```

Comment:
no need for the extra var, why not just `s.Namespaces[nsName] = &namespace{}`?

Reply:
right